internal/checker/cloudformation/rules/encryption/verify_es_encrypted_rest.rego

```diff
@@ -0,0 +1,57 @@
+# METADATA
+# title: "Verify that all information stored within Elasticsearch is securely encrypted while at rest."
+# description: "To protect your data, this feature encrypts everything stored in your domain, including search indexes, logs, temporary files, applications, and even backups, using secure keys managed by AWS. It uses the industry-standard AES-256 encryption algorithm for this purpose."
+# scope: package
+# related_resources:
+# - https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticsearch-domain.html
+# custom:
+#   id: CB_CFT_003
+#   severity: LOW
+package lib.cloudformation.CB_CFT_003
+import future.keywords.in
+
+resource := "AWS::Elasticsearch::Domain"
+
+is_valid {
+    some resources in input.Resources
+    resources.Type == resource
+}
+
+pass[properties] {
+    resources := input.Resources
+    is_valid
+    properties := resources.Properties
+    is_enabled := true
+    properties.Enabled == is_enabled
+}
+
```

Comment:
remove line 20 -26 block here.. since you only need to check EncryptionAtRestOptions that already cover in line 28-34